src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -93,6 +96,10 @@ public function newAction(Request $request)
             // See http://symfony.com/doc/current/book/controller.html#flash-messages
             $this->addFlash('success', 'post.created_successfully');
 
+            if ($form['createAndShow']->isClicked()) {
```

Comment:
using `$form->get('createAndShow')` instead of `$form['createAndShow']` looks similar to other Symfony's practices ... but I know nothing about the Form component, so ignore my comment :)